pelican/utils.py

```diff
@@ -51,79 +49,6 @@ def sanitised_join(base_directory, *parts):
     return joined
 
 
-def strftime(date, date_format):
```

Comment:
I have two reservations for simply removing this:
* It had extra functionality (see: #1446)
* It might be breaking if plugins/themes/etc were using these (similarly `SafeDatetime`)

Reply:
Ah, I got overzealous and figured it was unused after glancing at the docstring. Reverted the commit that removed these; I'll take another pass through it to just remove the Python 2 logic.